tests/Providers/Reviewsio/ServiceActionsTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace PodPoint\Reviews\Tests\Providers\ReviewsIo;
+
+use PodPoint\Reviews\ActionsInterface;
+use PodPoint\Reviews\ApiClientInterface;
+use PodPoint\Reviews\Providers\ReviewsIo\ServiceActions;
+use PodPoint\Reviews\Tests\TestCase;
+
+class ServiceActionsTest extends TestCase
+{
+    /**
+     * Constructor should be instance of APiClientInterface and it should assign to apiClient property.
+     */
+    public function testConstruct()
+    {
+        $action = new Mocked_ServiceActions($this->getMockedApiClient());
+
+        $this->assertInstanceOf(ActionsInterface::class, $action);
+        $this->assertInstanceOf(ApiClientInterface::class, $action->getApiClient());
+    }
+
+    /**
+     *  Making sure business id is setters and getters setting property as expected.
+     */
+    public function testStore()
+    {
+        $action = new ServiceActions($this->getMockedApiClient());
+
+        $store = 'foo-bar-123';
+        $action->setStore($store);
+
+        $this->assertEquals('foo-bar-123', $action->getStore());
+    }
+
+    /**
+     * Making sure the invite method to make sure it returns the response as expected.
+     *
+     * @return array|mixed
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testInvite()
+    {
+        $options = [
+            'name' => 'Customer Name',
+            'email' => 'customer@email.com',
+            'order_id' => 'order-number-321',
+        ];
+
+        $response = $this->getMockedResponse('{"status": "200", "message": "accepted"}');
+        $apiClient = $this->getMockedApiClient();
+        $apiClient->shouldReceive('sendRequest')->withAnyArgs()->andReturn($response);
+
+        $action = new ServiceActions($apiClient);
+        $action->setStore('store-number-123');
+
+        $inviteResponse = $action->invite($options);
+
+        $expectedResult = [
+            "status" => "200",
+            "message" => "accepted",
+        ];
+
+        $this->assertEquals($expectedResult, $inviteResponse);
+    }
+
+    /**
+     * Making sure the getReviews method to make sure it returns the response as expected.
+     *
+     * @return array|mixed
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testGetReviews()
```

Comment:
Can we add a test here where we check that there is no error (and that we get all reviews as expected) when we don't pass an argument to the getReviews() function?